script/cljdoc

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
-cd $(git rev-parse --show-toplevel)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_DIR/.." || exit
```

Comment:
This is so that the script also works when not ran inside a Git repo?

Reply:
Yes, that's correct.

It is convenient to run `ingest` from your library's git repo dir, and this change allows for that:

`../cljdoc/script/cljdoc ingest` ...